webroot/src/store/user/user.mutations.ts

```diff
@@ -115,4 +120,25 @@ export default {
         state.isLoadingPrivilegePurchaseOptions = false;
         state.error = error;
     },
+    [MutationTypes.SAVE_SELECTED_PRIVILEGE_PURCHASES_TO_STORE]: (
+        state: any,
+        privilegePurchaseChoices: Array<string>
+    ) => {
+        state.selectedPrivilegesToPurchase = privilegePurchaseChoices;
+    },
+    [MutationTypes.SET_ATTESTATIONS_ACCEPTED]: (state: any, areAttestationsAccepted: boolean) => {
+        state.arePurchaseAttestationsAccepted = areAttestationsAccepted;
+    },
+    [MutationTypes.POST_PRIVILEGE_PURCHASE_REQUEST]: (state: any) => {
+        state.isLoadingPrivilegePurchaseOptions = true;
+        state.error = null;
+    },
+    [MutationTypes.POST_PRIVILEGE_PURCHASE_SUCCESS]: (state: any) => {
+        state.isLoadingPrivilegePurchaseOptions = false;
+        state.error = null;
+    },
```

Comment:
Should this (or something related) also reset the `selectedPrivilegesToPurchase` & `arePurchaseAttestationsAccepted` values?